test/testUtils/TestSupport.scala

```diff
@@ -129,7 +129,8 @@ trait TestSupport extends UnitSpec with GuiceOneAppPerSuite with BeforeAndAfterE
     SessionKeys.lastRequestTimestamp -> "1498236506662",
     SessionKeys.authToken -> "Bearer Token"
   ).withHeaders(
-    HeaderNames.REFERER -> "/test/url"
+    HeaderNames.REFERER -> "/test/url",
+    "X-Session-ID" -> "xssession-12345"
```

Comment:
can you use a variable for ID here to keep it consistent?

Reply:
done